pkg/kv/kvnemesis/operations.proto

```diff
@@ -54,19 +54,28 @@ message ScanOperation {
 
 message PutOperation {
   bytes key = 1;
-  bytes value = 2;
+  uint32 Seq = 2 [(gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesisutil.Seq"];
   Result result = 3 [(gogoproto.nullable) = false];
 }
 
 message DeleteOperation {
   bytes key = 1;
-  Result result = 2 [(gogoproto.nullable) = false];
+  uint32 Seq = 2 [(gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesisutil.Seq"];
+  Result result = 3 [(gogoproto.nullable) = false];
 }
 
 message DeleteRangeOperation {
   bytes key = 1;
   bytes end_key = 2;
-  Result result = 3 [(gogoproto.nullable) = false];
+  uint32 Seq = 3 [(gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/kv/kvnemesis/kvnemesisutil.Seq"];
+  Result result = 4 [(gogoproto.nullable) = false];
+}
+
+message DeleteRangeUsingTombstoneOperation {
```

Comment:
nit: I do wonder if we should fold this into `DeleteRangeOperation`, given that they're the same KV method. But their semantics, especially from kvnemesis' perspective, are probably different enough that it's better to keep them separate.